frontend/src/scenes/data-management/events/DefinitionHeader.tsx

```diff
@@ -224,10 +229,7 @@ export function PropertyDefinitionHeader({
                 getFullDetailUrl: (propertyDefinition: PropertyDefinition) =>
```

Comment:
It felt odd to me to click on the property link and have it navigate to a list view that didn't show any more information about the property. Maybe we shouldn't do this link until we have additional data there?